torch/nn/functional.py

```diff
@@ -294,6 +294,27 @@ def lp_pool2d(input, norm_type, kernel_size, stride=None, ceil_mode=False):
     return out.mul(kw * kh).pow(1. / norm_type)
 
 
+def adaptive_max_pool1d(input, output_size, return_indices):
+    return _functions.thnn.AdaptiveMaxPool1d(output_size, return_indices)(input)
+
+
+def adaptive_max_pool2d(input, output_size, return_indices):
```

Comment:
Use the same default for `return_indices` that we have for other pooling functions. Also, can you add at least some very short descriptions of these modules, so they show up in the docs?